src/modules/Common/components/Datatables/UsernameTable.tsx

```diff
@@ -28,6 +34,24 @@ const UsernameTable = ({ exportName, data, onUsernameClick, nbData }: UsernameTa
       size: 1,
       Cell: ({ value }: any) => value.toLocaleString('en'),
     },
+    {
+      Header: 'Action',
+      align: 'right',
+      Cell: ({ ...rest }: any) => {
```

Comment:
`({ ...rest }:any)` is to get all left data
it is the same as `(rest: any)`
in this case we want `{ row }`